app/src/types/event.ts

```diff
@@ -134,6 +136,7 @@ export const parseEditEvent = ({
     hasWaitingList: hasWaitingList,
     isCancelled: isCancelled,
     isExternal: isExternal,
+    isInThePast: false,
```

Comment:
Denne er jeg litt usikker på. Jeg kan jo sikkert sjekke om parseEditDateTime(start) er større enn Datetime.now() eller no, men usikker på om det vits.

Reply:
Det er sikkert greit, man redigerer vel ikkje events in the past.